client/src/App.tsx

```diff
@@ -26,7 +27,8 @@ const App = (): JSX.Element => (
                   <Switch>
                     <Route exact path="/login" component={Login} />
                     <Route exact path="/signup" component={Signup} />
-                    <ProtectedRoute exact path="/dashboard" component={Dashboard} />
+                    <ProtectedRoute exact path="/dashboard/board/:id" component={Dashboard} />
```

Comment:
`Dashboard` component will need to be refactored if it'll only render on this path and fetch on load. For example, what happens when you go to `/dashboard/board/12312123123` where that `id` doesn't exist? `Dashboard` will try to render without having any data to display

Reply:
Since you're working on paths and what components to render, I suggest a `404` or `Not Found` page that'll show when an `id` for the board doesn't exist1

Reply:
@amahalwy , the user is redirected to the first board in this case.